src/components/SingleBlog/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly and matches the snapshot 1`] = `
+<h1>
+  loading
```

Comment:
well...either you or @jennethydyrova is gonna have to fix this one.

Reply:
still couldn't understand how to do it, in order to save time I'm gonna leave it to @jennethydyrova to do the test.

Reply:
I think you can take a look at the test I wrote and try to implement it here. If you will need help, I will be available tomorrow